Firestore/core/src/firebase/firestore/model/field_value.cc

```diff
@@ -58,11 +58,11 @@ bool Comparable(Type lhs, Type rhs) {
 
 }  // namespace
 
-FieldValue::FieldValue(const FieldValue& value) {
+FieldValue::FieldValue(const FieldValue& value) : tag_(Type::Null) {
```

Comment:
Can you add an in-class initializer to avoid having to repeat this?

```
class FieldValue {
  // ...
  Type tag_ = Type::Null;
};

Reply:
Good idea; done.